docs/samples/azure_synapse/product_recommendation_demo.ipynb

```diff
@@ -270,10 +234,10 @@
       "source": [
         "# Define sharable features using Feathr API\n",
         "\n",
-        "In this tutorial, we use Feathr Feature Store to help create a model that predicts users product rating. To make it simple, let's just predict users' rating for ONE product for an e-commerce website. (We have an [advanced demo](./product_recommendation_demo_advanced.ipynb) that predicts ratings for arbitrary products.)\n",
+        "In this tutorial, we use Feathr Feature Store and create a model that predicts users' product rating. To make it simple, let's just predict users' rating for ONE product for an e-commerce website. (We have an [advanced demo](./product_recommendation_demo_advanced.ipynb) that predicts ratings for arbitrary products.)\n",
```

Comment:
This advance nb link will be broken after this PR merged.

Reply:
Fixed, thanks for pointing it out.